Alloy/Renderers/SimpleGeometryRenderer.swift

```diff
@@ -32,20 +45,28 @@ public class SimpleGeometryRenderer {
                                         .makeRenderPipelineState(descriptor: renderPipelineStateDescriptor)
     }
 
+    // MARK: - Render
+
     public func render(geometry: MTLBuffer,
                        type: MTLPrimitiveType = .triangle,
                        fillMode: MTLTriangleFillMode = .fill,
                        indexBuffer: MTLIndexBuffer,
                        matrix: float4x4 = float4x4(diagonal: .init(repeating: 1)),
                        color: SIMD4<Float> = .init(1, 0, 0, 1),
                        using encoder: MTLRenderCommandEncoder) {
-        encoder.setVertexBuffer(geometry, offset: 0, index: 0)
-        encoder.set(vertexValue: matrix, at: 1)
-
-        encoder.set(fragmentValue: color, at: 0)
-
+        encoder.setVertexBuffer(geometry,
+                                offset: 0,
+                                index: 0)
+        encoder.set(vertexValue: matrix,
+                    at: 1)
```

Comment:
i personally think keeping `at` at the same line looks nicer, but not gonna argue here